packages/protocol-kit/tests/e2e/createTransaction.test.ts

```diff
@@ -195,17 +210,41 @@ describe('Transactions creation', () => {
           data: '0x',
           safeTxGas
         }
-        const safeTxData = await standardizeSafeTransactionData(
-          safeSdk.getContractManager().safeContract,
+        const safeTxData = await standardizeSafeTransactionData({
+          safeContract: safeSdk.getContractManager().safeContract as GnosisSafeContract,
           ethAdapter,
-          txDataPartial
-        )
+          tx: txDataPartial
+        })
         chai.expect(safeTxData.safeTxGas).to.be.eq(safeTxGas)
       }
     )
   })
 
   describe('createTransaction', async () => {
+    it('should create a single transaction with gasPrice=0', async () => {
+      const { predictedSafe, accounts, contractNetworks } = await setupTests()
+      const [account1, account2] = accounts
+      const ethAdapter = await getEthAdapter(account1.signer)
+      const safeSdk = await Safe.create({
+        ethAdapter,
+        predictedSafe,
+        contractNetworks
+      })
+      const safeTransactionData: SafeTransactionDataPartial = {
+        to: account2.address,
+        value: '500000000000000000', // 0.5 ETH
+        data: '0x',
+        baseGas: 111,
+        gasPrice: 0,
+        gasToken: '0x333',
+        refundReceiver: '0x444',
+        nonce: 555,
+        safeTxGas: 666
```

Comment:
baseGas, gasPrice and safeTxGas should be strings now

Reply:
Also the BASE_OPTIONS are now defined on top, so it's only necessary to define those that need to be changed. You can check an example on the next test